packages/gestalt/src/Box/Box.js

```diff
@@ -744,32 +744,32 @@ const PaddingPropType = PropTypes.oneOf([
 
 Box.propTypes = {
   children: PropTypes.node,
-  dangerouslySetInlineStyle: PropTypes.exact({
+  dangerouslySetInlineStyle: PropTypes.shape({
```

Comment:
Hey what was your reasoning for this change? Seems to make the interface more loose.

Reply:
@chrislloyd Flow isnt happy with that one (see https://github.com/pfarejowicz/gestalt/blob/8967902e3e96725c1c3113a8caf58acf7d566978/flow-typed/npm/prop-types_v15.x.x.js ). I see `PropTypes.exact` in the source code, but Facebook does not list it on the docs site (not sure why) (see https://github.com/facebook/prop-types ) so I think there is some miscommunication and a bug in flow-typed

Reply:
I don't think changing our code here is the right solution to get flow safety here. Are there other approaches?